docs/pages/enroll-resources/auto-discovery/databases/aws.mdx

```diff
@@ -39,141 +64,245 @@ teleport:
-  create and attach IAM policies.
-- In manual mode, Teleport prints the required IAM policies. You can then create
-  and attach them manually using the AWS management console.
-
-<Tabs>
-  <TabItem label="Automatic">
-  Either temporarily give IAM admin permissions to the host of the Discovery
-  Service or copy the service YAML configuration file to your desktop where you
-  have the IAM admin permissions.
-
-  Use this command to bootstrap the permissions automatically with YAML
-  configuration file of the Discovery Service:
-
-  ```code
-  $ teleport discovery bootstrap \
-    --attach-to-role arn:aws:iam::<Var name="aws-account-id"/>:role/<Var name="iam-role-name"/> \
-    --policy-name TeleportDatabaseDiscovery \
-    -c <Var name="/etc/teleport.yaml"/>
-  ```
-  </TabItem>
-
-  <TabItem label="Manual">
-  Use the following command to display the required IAM policies that you need
-  to create in your AWS console:
-
-  ```code
-  $ teleport discovery bootstrap --manual \
-    --attach-to-role arn:aws:iam::<Var name="aws-account-id"/>:role/<Var name="iam-role-name"/> \
-    --policy-name TeleportDatabaseDiscovery \
-    -c <Var name="/etc/teleport.yaml"/>
-  ```
-  </TabItem>
-</Tabs>
-
-<Details title="Bootstrapping with assume_role_arn in config">
-When `assume_role_arn` is configured for AWS matchers, `teleport discovery
-bootstrap` command determines the permissions required for the bootstrap target
-AWS IAM identity using the following logic:
-
-- When the target does not match `assume_role_arn` in any AWS matcher in the
-  configuration file, the target is assumed to be the Teleport Discovery
-  Service's AWS IAM identity and permissions are bootstrapped for all the AWS
-  matchers without `assume_role_arn`.
-- When an `--attach-to-role` target matches an `assume_role_arn` setting for
-  AWS matchers in the configuration file, permissions are bootstrapped only for
-  those AWS matchers.
-
-You will need to run the bootstrap command once with the Teleport Discovery
-Service's IAM identity as the policy attachment target, and once for each AWS
-IAM role that is used for `assume_role_arn`.
-</Details>
+(!docs/pages/includes/tctl-token.mdx serviceName="Discovery" tokenType="discovery" tokenFile="/tmp/token" !)
 
-## Step 4/4. Use the Discovery Service
+(!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
 ### Start the Discovery Service
 
 (!docs/pages/includes/start-teleport.mdx service="the Discovery Service"!)
 
-After the Discovery Service starts, database servers matching the tags and
-regions specified in the AWS section are added to the Teleport cluster
-automatically.
+## Step 4/8. Discovery group config
+
+Create a file <Var name="aws-example-discovery-config.yaml"/> and save it on a
+host where you can use `tctl`.
+
+(!docs/pages/includes/discovery/aws-db-discovery-config-picker.mdx!)
+
+Create the `discovery_config`:
+
+```code
+$ tctl create <Var name="aws-example-discovery-config.yaml"/>
```

Comment:
nit: should we mention somewhere that you can have multiple matchers per config?

Reply:
well it is sort of implied by the yaml list in the config, but we could call that out explicitly.

The thing is, if you select e.g. "RDS" in the pickers, then all of the pickers will change to RDS, so the guide becomes an RDS discovery guide.
If we mention you can use multiple matchers per config, then it becomes a guide to set up discovery for multiple types of db at the same time, which means they have to keep switching DB type from the dropdown picker to configure things in other steps or it won't work.
So I don't think I want to mention that.